suite/tests/api/ir_x86.c

```diff
@@ -81,8 +81,9 @@
 
 static byte buf[32768];
 
-static bool force_full_disp = false;
-static int memarg_disp = 0x37;
+#define DEFAULT_DISP 0x37
+#define SCALABLE_DISP 0x80
```

Comment:
This might seem to fit in 8-bit anyway: how about something larger.

Reply:
Good idea. I don't want to go too big, some scaled factors are only 2. 0x80 => 0x100 seems like a good idea.